### PR TITLE
SAK-492 refactoring

### PR DESCRIPTION
**Pull request by rafalkanski**
@filipgolonka @arkadiuszmalina @bartosz-kahl @wsromek

### CODE REVIEW

File: spec/Snt/Capi/Repository/ArticleRepositorySpec.php [hunk 1]
```diff
@@ -38,12 +41,21 @@ function it_finds_article_by_its_id(
...
 
-        $this->find(self::PUBLICATION_ID, self::ARTICLE_ID)->shouldBeArticleWithRawData([
+        $this->find(self::ARTICLE_ID)->shouldBeArticleWithRawData([
             'id' => 123,
             'title' => 'some text',
         ]);
     }
+    function it_throws_exception_when_can_not_fetch_response_using_http_client(
+        HttpClientInterface $httpClient
+    )
```
Comment (wsromek) [inline spec/Snt/Capi/Repository/ArticleRepositorySpec.php]: incorrect formatting?

File: spec/Snt/Capi/Repository/ArticleRepositorySpec.php [hunk 2]
```diff
@@ -38,12 +41,21 @@ function it_finds_article_by_its_id(
...
+    {
```
Comment (filipgolonka) [inline spec/Snt/Capi/Repository/ArticleRepositorySpec.php]: ) and { should be in one line

**Comment by arkadiuszmalina** [comment]
👍

File: src/Snt/Capi/Http/HttpClient.php [hunk 3]
```diff
@@ -37,10 +39,21 @@ public function __construct(
      */
     public function get($path)
     {
-        return $this->client
-            ->request('GET', $this->buildUri($path), $this->buildOptions())
-            ->getBody()
-            ->getContents();
+        try {
+            $response = $this->client->request(
+                'GET',
```
Comment (filipgolonka) [inline src/Snt/Capi/Http/HttpClient.php]: use const (?)

**Comment by wsromek** [comment]
👍

**Comment by filipgolonka** [comment]
:+1:

**Comment by barkahl** [comment]
👍

**Pull request merged**